ansible.cfg

```diff
@@ -6,7 +6,7 @@ pipelining=True
 host_key_checking=False
 gathering = smart
 fact_caching = jsonfile
-fact_caching_connection = /tmp
+fact_caching_connection = /tmp/deploy2
```

Comment:
is this intended or left over from testing?

Reply:
left over :(